README.md

```diff
@@ -19,24 +19,32 @@ This will hopefully put you in a position where your codebase will become more c
            phpcs-diff - detect violations based on a git diff
     
     SYNOPSIS
-           phpcs-diff [BASE_BRANCH]... [OPTION]...
+           phpcs-diff [-v] [--ruleset=PSR2] [BASE_BRANCH] [CURRENT_BRANCH]
     
     OPTIONS
            Here is a (very) short summary of the options available in phpcs-diff.
            
            -v
                   increase verbosity
+           --ruleset
+                  defines which php-cs ruleset to use (for example: PSR1, PSR12, ...)
                                    
 Basic example
 
 ```shell
-phpcs-diff develop -v
+phpcs-diff -v origin/develop develop
```

Comment:
I would rather this option be added as a feature, rather than a breaking change to existing functionality.

With your changes, we will have 3 variations:

1. `phpcs-diff` - Run a diff against unstaged changes.
2. `phpcs-diff origin/master` - Run a diff against the base branch (defined) and the current branch (determined by `git rev-parse --verify HEAD`)
3. `phpcs-diff origin/ master master` - Run a diff against the base branch and current branch (both defined)

My only ask here is to keep the functionality of 1.